ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -22,7 +24,7 @@ internal interface PaywallViewModel {
     val state: StateFlow<PaywallViewState>
 
     fun selectPackage(packageToSelect: Package)
-    fun purchaseSelectedPackage(activity: Activity)
+    fun purchaseSelectedPackage(context: Context)
```

Comment:
hmm, I worry about this becuase there's no indication to the developers that this should be a context with an `Activity`

Reply:
This is to fix the previews right? 🤔

Reply:
Yes, it's to fix the previews. Basically, it can't find an activity in the preview context so it fails... Any ideas of how to solve this? I can add a comment to the function so at least we are aware when using this function... And well, this context is usually an activity... I don't think `LocalContext.current` would return any service context or anything like that afaik.